clusterctl/examples/openstack/machines.yaml.template

```diff
@@ -0,0 +1,53 @@
+items:
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    generateName: openstack-master-
+    labels:
+      set: master
+  spec:
+    providerConfig:
+      value:
+        apiVersion: "openstackproviderconfig/v1alpha1"
+        kind: "OpenstackProviderConfig"
+        name: master
+        flavor: m1.medium
+        image: CC-Ubuntu16.04-20160610
+        availability_zone: nova
+        networks:
+        - uuid: ab14ce0d-5e1f-4e32-bf65-00416e3cc19c
+          floatingIP: 129.114.111.153
```

Comment:
does the user have to create the floating ip before they run clusterctl? (do we create it or just use it?)

Reply:
Yes, base on the logic we already have, user have to create this floating IP at first.
But we can do this work in MachineActuaor if we need.